packages/itwinui-css/src/file-upload/file-upload.scss

```diff
@@ -62,15 +64,150 @@
     }
   }
 
-  &.iui-drag > .iui-content {
-    background-color: var(--iui-color-background-accent-muted);
-    border-color: var(--iui-color-border-accent);
-    color: var(--iui-color-text-accent);
-    visibility: visible;
-
-    > svg,
-    .iui-template-icon {
+  &.iui-drag {
+    .iui-file-card-icon > svg,
+    .iui-file-card-empty-icon > svg {
       fill: var(--iui-color-icon-accent);
     }
+
+    > .iui-content {
+      visibility: visible;
+    }
+
+    > .iui-content,
+    .iui-file-card {
+      background-color: var(--iui-color-background-accent-muted);
+      border-color: var(--iui-color-border-accent);
+      color: var(--iui-color-text-accent);
+
+      > svg,
+      .iui-template-icon {
+        fill: var(--iui-color-icon-accent);
+      }
+    }
+  }
+}
+
+@mixin iui-file-card {
+  @include iui-reset;
+  @include iui-surface;
```

Comment:
Continuing conversation from above:

> > > FileUploadCard feels wrong because cards are usually elevated and have a background color. Maybe we should add those things?
> > 
> > if not elevation, then a border would also do.
> 
> Elevation and background color have been added

Shouldn't empty state also have some kind of border? I'll defer to @FlyersPh9 for final decision on what the UI should look like.

Reply:
Paired with Adam and added border.

Reply:
@FlyersPh9 Can you confirm the visuals before this is merged? I'm looking at the linked issue and the example screenshot there has no elevation - it might be used as part of a bigger card.

Reply:
also in the linked issue, the whole background/border is blue after file is uploaded, whereas in this PR, it's regular surface colors. not sure which one we want.

Reply:
@adamhock can you remove the drop shadow & leave everything else as is?

Reply:
The drop shadow has been removed